Gemfile

```diff
@@ -1,77 +1,102 @@
 source 'https://rubygems.org'
 ruby '2.4.3'
 
+# Standard rails
 gem 'rails', '>= 5.1'
+gem 'puma' # roar
```

Comment:
Just noting that puma moved from prod only to all. If that change was intentional, then all looks GTG!

Reply:
Yep, this is intentional! There was an effort awhile back to test in an environment as close to actual production as possible in rails, and part of that is 'your prod server gem should be your dev server gem too', so now we're using puma in dev instead of webbrick.